vlsi/example-vlsi

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+
+import hammer_vlsi
+from hammer_vlsi import CLIDriver
+
+from typing import Dict, Callable, Optional
+
+def example_place_tap_cells(x: hammer_vlsi.HammerTool) -> bool:
+    x.append('''
+# TODO
+''')
+    return True
+
+class ExampleDriver(CLIDriver):
+
+    def action_map(self) -> Dict[str, Callable[[hammer_vlsi.HammerDriver, Callable[[str], None]], Optional[dict]]]:
```

Comment:
This isn't the right way to simply add a step.

Reply:
This is all taken from @jwright6323 's intech22 branch of the old project-template

Reply:
🔥

Reply:
You just overload `get_extra_par_hooks`
```
def get_extra_par_hooks(self) -> List[HammerToolHookAction]:
  return [hammer_vlsi.HammerTool.make_replacement_hook("place_tap_cells", example_place_tap_cells)]
```